docs/examples/plot_interval_transposition_error.py

```diff
@@ -0,0 +1,157 @@
+"""
+Modeling with interval averages
+===============================
+
+Transposing interval-averaged irradiance data
+"""
+
+# %%
+# This example shows how not accounting for the difference between
+# instantaneous and interval-averaged time series data can introduce
+# error in the modeling process. An instantaneous time series
+# represents discrete measurements taken at each timestamp, while
+# an interval-averaged time series represents the average value across
+# each data interval.  For example, the value of an interval-averaged
+# hourly time series at 11:00 represents the average value between
+# 11:00 and 12:00 (if the series is left-labeled), the average value
```

Comment:
I think it's important to remind readers that the left-labeled interval is inclusive of the left label and exclusive of the right label. So in this example inclusive of 11:00 and exclusive of 12:00. And the opposite for the right labeled interval. Might want to break up the sentences to incorporate that concept.